src/SqlServerCache/Program.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Data;
+using System.Data.SqlClient;
+using Microsoft.Framework.Logging;
+using Microsoft.Framework.Runtime.Common.CommandLine;
+
+namespace SqlServerCache
```

Comment:
@Eilon Didn't we decide to change all the commands packages to Microsoft.* rather than having a flat name?